src/stress_scheduler.py

```diff
@@ -27,66 +27,66 @@
 
 ### Start the stresses on the various resources
 
-def start_causal_cpu(ssh_client, disk_rate, network_rate):
-    throttle_network(ssh_client, network_rate)
-    return throttle_disk(ssh_client, disk_rate)
+def start_causal_cpu(ssh_client, container_id, disk_rate, network_rate):
```

Comment:
For all the functions here, keep in mind that that just passing in the argument container_id to this helper stress functions probably makes that function redundant. I'll accept it for now but your next PR will probably need to change this.